lab1.js

```diff
@@ -99,12 +96,22 @@ assert(sentence2 === 'chirp chirp chirp chirp chirp chirp chirp chirp chirp.',
 */
 
 var favoriteAnimals = ['elephant', 'penguin', 'eagle', 'camel'];
-var nextAnimal;
 
-// TODO: 12 points
-// Assign one of your favorite animals to nextAnimal using Math.random() to pick
+var nextAnimal = Math.random();
 
-// your code goes here
+if (nextAnimal < 0.25) {
+  nextAnimal = favoriteAnimals[0];
```

Comment:
this approach works...and you can make it cleaner. Check out this approach: http://stackoverflow.com/questions/5915096/get-random-item-from-javascript-array